API/Program.cs

```diff
@@ -18,6 +19,14 @@ public static async Task Main(string[] args)
             var host = CreateHostBuilder(args).Build();
             using var scope = host.Services.CreateScope();
             var services = scope.ServiceProvider;
+
+            Log.Logger = new LoggerConfiguration()
```

Comment:
Can register the log as Singleton in the Startup.cs, so that it's managed by the Asp.net Core IoC

Reply:
Also, all the log configurations should be written into configuration files, so that for different environments, you can have different settings. Such as, on your local machine, you can use Seq as the sink, and after you deploy the app to AWS/Azure, you can point the log sink to use Cloudwatch or ApplicationInsights, or even some other Logging system, such as ELK and Splunk